pinot-controller/src/test/java/org/apache/pinot/controller/helix/core/periodictask/ControllerPeriodicTaskTest.java

```diff
@@ -128,7 +130,8 @@ public void testControllerPeriodicTaskCalls() {
     assertTrue(_processTablesCalled.get());
     assertEquals(_tablesProcessed.get(), _numTables);
     assertEquals(
-        _controllerMetrics.getValueOfGlobalGauge(ControllerGauge.PERIODIC_TASK_NUM_TABLES_PROCESSED, TASK_NAME),
+        MetricValueUtils.getGaugeValue(_controllerMetrics,
```

Comment:
Should we add a function named MetricValueUtils.getTaskGaugeValue(), seems like a common use case over controller code.

Reply:
thanks for the feedback!